rest/src/main/java/io/syndesis/rest/v1/handler/user/UserHandler.java

```diff
@@ -15,30 +15,49 @@
  */
 package io.syndesis.rest.v1.handler.user;
 
+import javax.ws.rs.GET;
 import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.xml.ws.Response;
 
 import io.syndesis.dao.manager.DataManager;
 import io.syndesis.model.Kind;
 import io.syndesis.model.user.User;
+import io.syndesis.openshift.OpenShiftService;
 import io.syndesis.rest.v1.handler.BaseHandler;
 import io.syndesis.rest.v1.operations.Getter;
 import io.syndesis.rest.v1.operations.Lister;
 import io.swagger.annotations.Api;
 
+import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.stereotype.Component;
+import org.springframework.util.Assert;
 
 @Path("/users")
 @Api(value = "users")
 @Component
 public class UserHandler extends BaseHandler implements Lister<User>, Getter<User> {
 
-    public UserHandler(DataManager dataMgr) {
+    private OpenShiftService openShiftService;
+
+    public UserHandler(DataManager dataMgr, OpenShiftService openShiftService) {
         super(dataMgr);
+        this.openShiftService = openShiftService;
     }
 
-     @Override
+    @Override
     public Kind resourceKind() {
         return Kind.User;
     }
-}
 
+    @Path("~")
```

Comment:
What does this kind of path mean ?

Reply:
Thats just a static path to "~".  I like it since ~ has a similar meaning on the CLI.

Reply:
Showoffs

Reply:
Copied from openshift... I like it because it's a special character that is highly unlikely to be in anyone's username and not too annoying to disallow for users that might want to create their own usernames.